resources/views/info/summary.html

```diff
@@ -1,13 +1,13 @@
 <div class="yii-debug-toolbar__block">
     <a href="#" title="">PHP
-        <span class="yii-debug-toolbar__label" data-key="phpVersion">0</span>
+        <span class="yii-debug-toolbar__label" data-key="php.version">0</span>
     </a>
 </div>
 <script>
     (function () {
         setTimeout(() => {
-            document.querySelector('[data-key=phpVersion]').textContent = YiiDebug.currentSession.phpVersion || 'n/a';
-            document.querySelector('[data-key=phpVersion]').parentElement.href = YiiDebug.viewerUrl + '/info';
+            document.querySelector('[data-key="php.version"]').textContent = YiiDebug.currentSession.web.php.version || 'n/a';
```

Comment:
php.version or web.php.version?

Reply:
`php.version` is common for `web.php.version` or `console.php.version` and etc. I'll add `|| console.php.version` a bit later